wandb/sdk_py27/wandb_artifacts.py

```diff
@@ -202,6 +168,19 @@ def add_manifest_file(log_phy_path):
         termlog("Done. %.1fs" % (time.time() - start_time), prefix=False)
 
     def add_reference(self, uri, name=None, checksum=True, max_objects=None):
+        """adds `uri` to the artifact via a reference, located at `name`. 
+        You can use Artifact#get_path(`name`) after downloading
```

Comment:
Technically, you can call `get_path('name')` without needing to first call `download()`.

Reply:
true - corrected (in next commit)